docs/roadmap.md

```diff
@@ -8,7 +8,7 @@
 
 !!! info "Key <br> :white_check_mark: - Completed <br> :tools: - In active development <br> :construction: - Queued <br>:pencil: - Yet to be scoped"
 
-On The Horizon
+Beyond 2024
```

Comment:
I should have picked this up previously but the items on the roadmap table don't align. E.g.

<img width="842" alt="image" src="https://github.com/IDinsight/aaq-core/assets/14125957/6d131841-f0ce-4d7a-8766-e19ff3e31f07">

Or rather, it is dependent on screen size. Can you have each as a row  to make it render correctly?

Reply:
Fixed!